src/components/ConnectedMap/ConnectedMap.tsx

```diff
@@ -3,26 +3,9 @@ import mapLibre from 'maplibre-gl';
 import { useAction, useAtom } from '@reatom/react';
 import { currentMapAtom, mapListenersAtom } from '~core/shared_state';
 import { layersOrderManager } from '~core/logical_layers/utils/layersOrder/layersOrder';
+import config from '~core/app_config';
 import Map from './map-libre-adapter';
-import DeckGl from './deck-gl';
```

Comment:
should we remove this one? I can't see this file ever used somewhere

Reply:
nice catch!